src/Workload/Microsoft.Maui.Controls.Sdk/Sdk/WinUI.SingleProject.targets

```diff
@@ -0,0 +1,6 @@
+<!-- Workarounds for WinUI when $(SingleProject) is true -->
+<Project>
+  <!-- Required - Overwrite tasks that are not needed when multitargeting -->
+  <Target Name="ValidateWinUIPlatform" />
+  <Target Name="BinPlaceBootstrapDll" />
```

Comment:
Does this come in late enough? The WinUI targets from the nugets create this target, and we have to overwrite with a dummy.

Reply:
This is imported as late as a workload can do it:
https://github.com/dotnet/maui/blob/f95c2e5c9a6e19817c75ddb93fae98ca8739399f/src/Workload/Microsoft.Maui.Controls.Sdk/Sdk/Microsoft.Maui.Controls.Sdk.After.targets#L9

https://github.com/dotnet/maui/blob/f95c2e5c9a6e19817c75ddb93fae98ca8739399f/src/Workload/Microsoft.Maui.Controls.Sdk/Sdk/Microsoft.Maui.Controls.Sdk.targets#L14-L16

We'd need to test, but if it doesn't work we'd have to put the `Directory.Build.targets` back.